File: debug_string.py

path = r"c:\My Web Sites\letrafluidacopy\letrafluida.com.br\assets\index-BY5pObJn.js"
with open(path, "r", encoding="utf-8") as f:
    content = f.read()

keyword = "garrancho"
if keyword in content:
    idx = content.find(keyword)
    start = max(0, idx - 100)
    end = min(len(content), idx + 100)
    snippet = content[start:end]
    print(f"FOUND at {idx}")
    print("SNIPPET_RAW:")
    print(repr(snippet))  # repr mostra \n, \t e aspas reais
else:
    print("KEYWORD NOT FOUND")

keyword2 = "Beatriz Souza"
if keyword2 in content:
    idx = content.find(keyword2)
    start = max(0, idx - 50)
    end = min(len(content), idx + 200)
    snippet = content[start:end]
    print(f"FOUND DEPOIMENTO at {idx}")
    print("SNIPPET_DEPOIMENTO:")
    print(repr(snippet))
else:
    print("DEPOIMENTO NOT FOUND")

File: debug_string.js
const fs = require('fs');
const path = 'c:\\My Web Sites\\letrafluidacopy\\letrafluida.com.br\\assets\\index-BY5pObJn.js';
try {
    const content = fs.readFileSync(path, 'utf8');

    // Check 1: Garrancho
    const idx = content.indexOf('garrancho');
    if (idx >= 0) {
        console.log("--- HERO RAW ---");
        console.log(JSON.stringify(content.substring(idx - 100, idx + 100)));
    } else {
        console.log("HERO NOT FOUND");
    }

    // Check 2: Beatriz Souza
    const idx2 = content.indexOf('Beatriz Souza');
    if (idx2 >= 0) {
        console.log("--- DEPOIMENTO RAW ---");
        console.log(JSON.stringify(content.substring(idx2 - 50, idx2 + 200)));
    } else {
        console.log("DEPOIMENTO NOT FOUND");
    }

    // Check 3: Letra Bonita e Rápida
    const idx3 = content.indexOf('Letra Bonita e Rápida');
    if (idx3 >= 0) {
        console.log("--- BENEFICIO RAW ---");
        console.log(JSON.stringify(content.substring(idx3 - 50, idx3 + 50)));
    }

} catch (e) { console.error(e); }
